### ISSUE TITLE
[Support] How to make the mod work

### ISSUE THREAD

**Issue by Anonyku05** [LitnhJacuzzi/CursorCentered-Fix#7]
Hi, i'm not that good in Linux, wanted to know how to make the mod work (Debian 12, KDE, Wayland), i installed ydotoold using apt install but don't know where to go from here

**Comment by LitnhJacuzzi** [comment]
Have you also completed steps provided in [this version](https://modrinth.com/mod/cursorcentered-fix/version/1.2-alpha)'s changelog? If yes the cursor should be centered normally.

**Comment by Anonyku05** [comment]
Yes i did, here is a screenshot of my /etc/sudoers file (my username is "v"):
![image](https://github.com/user-attachments/assets/7686212c-f5e7-4da2-9da0-86ea24dba83a)

The mod is installed but not working
Do i need to make something manually in order to make it work ?

**Comment by Anonyku05** [comment]
Edit: The mod is "working" but the cursor is not centered, it ends up in the corner of my screen

**Comment by LitnhJacuzzi** [comment]
> Edit: The mod is "working" but the cursor is not centered, it ends up in the corner of my screen

Does it mean "the cursor always appears at the corner of your screen after opening a GUI"?
If yes, try execute `sudo ydotool mousemove <anyX> <anyY>` in your terminal to see whether the cursor could be moved correctly.

**Comment by Anonyku05** [comment]
Sorry for late reponse, but to answer your question: yes it is
Here's what happened when i tried typing the command you just gave me:
![image](https://github.com/user-attachments/assets/53406f14-e3bf-4956-a71b-6242fdfc4c06)

Still same issue

**Comment by LitnhJacuzzi** [comment]
`anyX` and `anyY` represents an arguement name and you should replace them with specific numbers. Such as `sudo ydotool mousemove 100 100`, if the cursor is moved to coordinate (100,100) then the backend tool works normally.
